source/Calamari.Common/Features/Scripting/Bash/BashScriptBootstrapper.cs

```diff
@@ -116,6 +116,7 @@ public static (string bootstrapFile, string[] temporaryFiles) PrepareBootstrapFi
                 writer.NewLine = LinuxNewLine;
                 writer.WriteLine("#!/bin/bash");
                 writer.WriteLine("source \"$(pwd)/" + Path.GetFileName(configurationFile) + "\"");
+                writer.WriteLine("shift");
```

Comment:
Maybe a comment (if it is what we do here) about why shift is needed to be added?

Reply:
✅ Comment added